Documentation/concepts/scalability/identity-relevant-labels.rst

```diff
@@ -20,21 +20,22 @@ application.
 
 By default, Cilium evaluates the following labels:
 
-=================================== ==================================================
-Label                               Description
------------------------------------ --------------------------------------------------
-``k8s:io.kubernetes.pod.namespace`` Include all ``io.kubernetes.pod.namespace`` labels
-``k8s:app.kubernetes.io``           Include all ``app.kubernetes.io`` labels
-``k8s:!io.kubernetes``              Ignore all ``io.kubernetes`` labels
-``k8s:kubernetes.io``               Ignore all other ``kubernetes.io`` labels
-``k8s:beta.kubernetes.io``          Ignore all ``beta.kubernetes.io`` labels
-``k8s:k8s.io``                      Ignore all ``k8s.io`` labels
-``k8s:!pod-template-generation``    Ignore all ``pod-template-generation`` labels
-``k8s:!pod-template-hash``          Ignore all ``pod-template-hash`` labels
-``k8s:!controller-revision-hash``   Ignore all ``controller-revision-hash`` labels
-``k8s:!annotation.*``               Ignore all ``annotation labels``
-``k8s:!etcd_node``                  Ignore all ``etcd_node`` labels
-=================================== ==================================================
+====================================== ==================================================
+Label                                  Description
+-------------------------------------- --------------------------------------------------
+``k8s:io.kubernetes.pod.namespace``    Include all ``io.kubernetes.pod.namespace`` labels
+``k8s:io.cilium.k8s.namespace.labels`` Include all ``io.cilium.k8s.namespace.labels`` labels
```

Comment:
for reviewer, this one is added

Reply:
This one is not entirely correct. Cilium does not evaluate this label from the pod spec, this label is generated from the labels of the namespace where the pod is running.

Reply:
oh I see, I was thinking to mention this one for the completeness. But as per your comment, this is internally generated labels, which will not have any values from user point of view.

Let me remove this. Thanks.